nilearn/plotting/find_cuts.py

```diff
@@ -210,7 +210,15 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
     data = _smooth_array(data, affine, fwhm='fast')
 
     if spacing == 'auto':
-        spacing = max(int(.5 / n_cuts * data.shape[axis]), 1)
+        if n_cuts > 0.:
+            spacing = max(int(.5 / n_cuts * data.shape[axis]), 1)
+        else:
+            message = ("If the input 'cut_coords' is an integer, it only "
```

Comment:
I think your message is relevant to cut_coords in plot_* (e.g. plot_stat_map) but not really n_cuts in find_cut_slices ...